.gitignore

```diff
@@ -1,3 +1,4 @@
 .vscode
 /coverage
 /node_modules
+.dccache
```

Comment:
As a result of security suggestion to use the Snyk VS Code extension, it creates this file locally so wanted to add it to gitignore